php7/server.php

```diff
@@ -58,28 +58,33 @@
             return new Response(500, [], "$codePath not found");
```

Comment:
@checkin247 can you please update version https://github.com/fission/environments/blob/master/php7/envconfig.json file ? So that once we merge PR. New environment image is updated.

Reply:
@sanketsudake [increase environment version to 1.31.2](https://github.com/fission/environments/pull/292/commits/bdc86929f548f2889cadc05c56c1e2120fab5f73)